Modules/posixmodule.c

```diff
@@ -176,14 +176,6 @@ corresponding Unix manual entries for more information on calls.");
 #else
 /* Unix functions that the configure script doesn't check for */
 #define HAVE_EXECV      1
-/* bpo-32705: Current Android does not have posix_spawn
- * Most likely posix_spawn will be available in next Android version (Android
- * P, API 28). Need revisit then. See
- * https://android-review.googlesource.com/c/platform/bionic/+/504842
- **/
-#ifndef __ANDROID__
-#define HAVE_POSIX_SPAWN 1
```

Comment:
just for my own info, why do you remove this condition? you won't support all the versions of Android... is it right?

Reply:
The check is done now in the configuration phase and it will be automatically detected if the system has this function.